src/FS-FileDirectory-Adapters.package/FSFileDirectoryAdapter.class/instance/deleteDirectory..st

```diff
@@ -0,0 +1,3 @@
+as yet unclassified
+deleteDirectory: aString
+	(reference resolve: aString) delete.
```

Comment:
Should this fail if the directory is not empty or should it delete all of the contents recursively?

If the latter, you must use deleteAll.

Reply:
So at least on Windows, `FileDirectory >> #deleteDirectory:` fails if the directory is not empty, so this implementation should be correct. Anyway, I will add `#recursiveDelete` and map it to `#deleteAll`.